test_docket_and_docket_entry_already_exist is randomly failing due to a change in Django's get_available_name

The test test_docket_and_docket_entry_already_exist is randomly failing:

`AssertionError: 'gov.uscourts.scotus.asdf.1.0' not found in 'recap/gov.uscourts.scotus.asdf/gov_aWMPDJD.uscourts.scotus.asdf.1.0.pdf'`

The issue here is due to a file name collision, so Django appends a random prefix to the file name. Previously, it was something like:

`gov.uscourts.scotus.asdf.1.0_aWMPDJD.pdf`

But now it’s:

`gov_aWMPDJD.uscourts.scotus.asdf.1.0.pdf`

After these changes:
https://github.com/django/django/commit/eb2d49b73408f1e60b8e1fa7fee822816945e1cd#diff-2f8524035203a28c596eb210d5b886a12d5a46215a649bee85f9987cc335d128

I’m wondering if we should just tweak the test to accept this new format, or if we should somehow preserve the old format for existing files?

--- a/cl/recap/tests.py
+++ b/cl/recap/tests.py
@@ -1485,7 +1485,8 @@ class RecapPdfTaskTest(TestCase):
         Alas, we fail. In theory, this shouldn't happen.
         """
         self.de.delete()
-        rd = async_to_sync(process_recap_pdf)(self.pq.pk)
+        with mock.patch("cl.recap.tasks.asyncio.sleep"):
+            rd = async_to_sync(process_recap_pdf)(self.pq.pk)
         self.assertIsNone(rd)
         self.pq.refresh_from_db()
         # Confirm PQ values.
@@ -1506,7 +1507,9 @@ class RecapPdfTaskTest(TestCase):
         self.assertTrue(rd.is_available)
         self.assertTrue(rd.sha1)
         self.assertTrue(rd.filepath_local)
-        self.assertIn("gov.uscourts.scotus.asdf.1.0", rd.filepath_local.name)
+        file_name = rd.filepath_local.name.split("/")[2]
+        self.assertIn("gov", file_name)
+        self.assertIn("uscourts.scotus.asdf.1.0", file_name)
 
         mock_extract.assert_called_once()
 
@@ -1524,7 +1527,8 @@ class RecapPdfTaskTest(TestCase):
         In practice, this shouldn't happen.
         """
         self.docket.delete()
-        rd = async_to_sync(process_recap_pdf)(self.pq.pk)
+        with mock.patch("cl.recap.tasks.asyncio.sleep"):
+            rd = async_to_sync(process_recap_pdf)(self.pq.pk)
         self.assertIsNone(rd)
         self.pq.refresh_from_db()
         # Confirm PQ values.

